sites/default/config/field.field.node.article.field_paragraph.yml

```diff
@@ -0,0 +1,50 @@
+uuid: 66152656-0291-4c22-9994-805062079bac
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_paragraph
+    - node.type.article
+  module:
+    - entity_reference_revisions
+id: node.article.field_paragraph
+field_name: field_paragraph
+entity_type: node
+bundle: article
+label: Paragraph
```

Comment:
I'm fine with the field name for this but I think the label could be better. Paragraph confuses me as the end user I think. Thoughts?

Reply:
Fixed it to be more inline with what we've done previously (see above comment).